websites/N/Neko-sama/presence.ts

```diff
@@ -66,7 +66,7 @@ presence.on("UpdateData", async () => {
 						}`;
 						presenceData.largeImageKey =
 							episodeImage ===
-							"https://cdn.rcd.gg/PreMiD/websites/N/Neko-sama.fr/assets/0.png"
+							"https://neko-sama.fr/images/default_thumbnail.png"
```

Comment:
this image will just be uploaded again to the cdn as a square 512x512 image.

Reply:
Okay but in the program, this image link is used to make an equality, if the link change the equality will always be false (and this is why image don't showing for default image episode in the presence) because I don't need an image here but the string of the image. Is there anything I can do to avoid the link be changed ?

Reply:
uhmmmm, you could make it a template string, cause those are ignored.
have a const websiteDomain = https://neko-same.fr
and then have the === as `${websiteDomain}/images...`

Reply:
Works well, thanks.